project_generation/content/templates/api/api/api.go.tmpl

```diff
@@ -0,0 +1,27 @@
+package api
```

Comment:
I believe this is because these are the `api` project type templates for the `api` package.

Reply:
Yes, this is the case. The eventual aim is to no longer need the manifest because the templates dir structure will drive the generation. In that scenario, an api project would contain everything from `generic` +`base-app` + `api` directories.